### DISCUSSION TITLE
Can I save a page without CSS and/or JS?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [gildas-lormeau/SingleFile#1493] [Q&A] [answered]
Sometimes, I want to copy a page's content into a Discourse or Git(Hub/Lab) post or comment, which have limited sizes.

**Comment by gildas-lormeau** [comment, marked as answer]
This should be possible by enabling/disabling the settings `Network > blocked resources` in the options page of SingleFile.
